File: commands/misc/autoremind.js
const { SlashCommandBuilder } = require('discord.js');

module.exports = {
	data: new SlashCommandBuilder()
		.setName('autoremind')
		.setDescription('config the leaf claim autoreminding')
    .addIntegerOption(option =>
		  option.setName('config')
			.setDescription('-1 for no reminding, otherwise reminds you x seconds before its avaliable (so 0 for no wait time)')
      .setMinValue(-1)
      .setRequired(true)),
	execute(client, interaction) {
	  interaction.reply({ content: 'settings changed', ephemeral: true }).catch(e=>{})
    let config = interaction.options.getInteger('config');
    let sett = JSON.parse(client.db.v2getuser('settings',interaction.user.id,'{"autoremind":{"time":300}}'));
    sett.autoremind.time = config;
    client.db.v2setuser('settings',interaction.user.id,JSON.stringify(sett));
  }
};

File: commands/misc/autoremind.py
import discord
import json

@discord.app_commands.command(name="autoremind", description='config the leaf claim autoreminding')
@discord.app_commands.describe(config="-1 for no reminding, otherwise reminds you x seconds before its avaliable (so 0 for no wait time)")
async def autoremind(interaction: discord.Interaction, config:int):
    if config < -1:
        return
    try:
        await interaction.response.send_message('settings changed', ephemeral=True)
    except:
        pass
    sett = json.loads(interaction.client.db.v2getuser("settings", interaction.user.id, str({"autoremind":{"time":300}})))
    sett.autoremind.time = config
    interaction.client.db.v2setuser("settings", interaction.user.id, json.dumps(sett))
